prog/setup_hugepages.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class Prog::SetupHugepages < Prog::Base
+  subject_is :sshable, :vm_host
+
+  def start
+    hugepage_size = "1G"
+    # TODO: how should we calculate required hugepages?
```

Comment:
Although this may expose fragmentation issues, I would use something like:

`((cores - 1) / cores) * Physical RAM  + 1GB (for SPDK) `

Qualitatively, this scheme reserves one core worth of memory, and 1GB of that is for SPDK via the +1.

Reply:
I went with the formula you suggested for now. We'll try it on few servers & update it to cover all cases.